zdm/scripts/test_MCMC2_single.py

```diff
@@ -0,0 +1,92 @@
+"""
```

Comment:
can we move all of these tests elsewhere?

`scripts/` is intended to hold useful scripts. :)